charts/k8tz/templates/controller.yaml

```diff
@@ -81,6 +92,27 @@ spec:
               scheme: HTTPS
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+        - name: {{ .Chart.Name }}-cert-watcher
+          args:
+          - "cert-watcher"
+          - "--tls-crt"
+          - "/run/secrets/shared-tls/tls.crt"
+          - "--tls-key"
+          - "/run/secrets/shared-tls/tls.key"
+          - "--secret-name"
+          - {{ include "k8tz.fullname" . }}-tls
+          - "--secret-namespace"
+          - {{ .Values.namespace }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
```

Comment:
note for you and myself: security context source is going to be changed as part of #58, and will be `include "k8tz.podSecurityContext" .` instead of directly from the values